lib/parse.js

```diff
@@ -779,9 +810,12 @@ function parse($TEXT, options) {
         handle_regexp();
         switch (S.token.type) {
           case "string":
-            var dir = S.in_directives, stat = simple_statement();
-            // XXXv2: decide how to fix directives
-            if (dir && stat.body instanceof AST_String && !is("punc", ",")) {
```

Comment:
Yeah, there is a check needed to make sure a semicolon is following, which may be there already or auto-inserted